### PR TITLE
Enable flag for custom footer string

### PR DESCRIPTION
**Pull request by lvlds**
I wanted to use the footer flag, so I added it to the entrypoint script.
It might be useful for someone.

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -12,5 +12,6 @@ services:
       OAUTH2_PROXY_UPSTREAM: "http://...:.../"
       OAUTH2_PROXY_CLIENT_ID: "...apps.googleusercontent.com"
       OAUTH2_PROXY_CLIENT_SECRET: "..."
+      OAUTH2_PROXY_FOOTER: "Secured with Docker OAuth2 Proxy"
```
Comment (jodok) [inline docker-compose.yaml]: i'd prefer to not have it in the sample docker-compose.yaml

**Comment by lvlds** [comment]
Hi, I removed the footer env var from docker compose as requested.
Thanks for the feedback.

**Comment by lvlds** [comment]
Will do

**Comment by lvlds** [comment]
I only added in the list of env vars, it does not have a description because the others don't have.

**Pull request merged**